core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -16171,4 +16175,24 @@ protected void handleGetNonExistentUser(String userName, String profileName) thr
         handleGetUserClaimValuesFailure(errorCode, errorMessage, userName, null, profileName);
         throw new UserStoreException(errorCode + " - " + errorMessage);
     }
+
+    /**
+     * The password validity timeout value is set by server configuration value from carbon.xml file.
+     * If value is not present the default value of 24 hours is returned.
+     * @return password validity timeout in hours.
+     */
+    protected int getDefaultPasswordValidityPeriodInHours() throws UserStoreException {
```

Comment:
Can't we make this method a static? I don't see it using any instance variables

Reply:
done